style.css

```diff
@@ -313,26 +371,306 @@ h4{
   align-items: center;
 }
 
+.background-section {
+  background: #fff;
+}
+
 .contact-section {
-  background: #6070FF;
-  background-image: url(src/img/contact\ form\ shapes-mobile@2x.png);
+  background: #6070ff;
+  background-image: url(src/img/contact\ form\ shapes-mobile@2x.svg);
   background-repeat: no-repeat;
   background-size: 55%;
   background-position-x: right;
-  background-position-y: -25%;
+  background-position-y: 2.5em;
   border-top-left-radius: 4rem;
 }
 
 .contact-text-section {
   display: flex;
   grid-column: 1/9;
   flex-direction: column;
-  border: 1px solid #dfe1e6;
-  border-top-right-radius: 4em;
+  align-items: center;
 }
 
-.contact-title h4{
-  margin: 2.5em 0.6em 0 0.6em;
+.contact-title h4 {
+  margin: 3em 0.6em 0 0.6em;
   font-size: 2.5em;
   color: #fff;
-}
\ No newline at end of file
+}
+
+form {
+  display: flex;
+  grid-column: 1/9;
+  flex-direction: column;
+  align-items: left;
+  margin: 0.6em 3em 0 0.6em;
+}
+
+.input-box input {
+  width: 100%;
+  border: 1px solid #cfd8dc;
+  border-radius: 0.5rem;
+  margin: 0.6em 1em 0 0.6em;
+  text-align: left;
+  padding: 3%;
+}
+
+.input-box textarea {
+  width: 100%;
+  border: 1px solid #cfd8dc;
+  border-radius: 0.5rem;
+  margin: 0.6em 0.6em 0 0.6em;
+  text-align: left;
+  padding: 3%;
+}
+
+button {
+  background-color: #fff;
+  padding: 3%;
+  font-weight: 700;
+  font-size: 1.06em;
+  color: #6070ff;
+  border: 1px solid #cfd8dc;
+  border-radius: 0.5rem;
+  cursor: pointer;
+  margin: 1em 0.6em 2.5em 0.6em;
+}
+
+@media (min-width: 768px) {
+  body {
+    font-size: 20px;
+  }
+
+  .logo {
+    margin: 0 2% 0 70%;
+  }
+
+  .nav-right img {
+    display: none;
+  }
+
+  .flex-div ul {
+    display: flex;
+    justify-content: space-between;
+    margin: 1em 0 0.8em 60%;
+  }
```

Comment:
- [ ] Kindly stick with the *Figma* design as closely as possible, it seems like your design is not matching with original. Please have a look at the below comparison table to understand the problem in a better way through visualization, according to which your **navigation** font-size and weight not matching.

###  Design Comparison

<table>
<tr>
<th>Your Design</th>
<th>Figma Design</th>
</tr>

<tr>
<td>

![Screenshot from 2023-06-10 13-27-25](https://github.com/JhonatanErosa/Project-2_Portfolio/assets/66476329/663019fa-62b2-44c1-a89f-1dd085759448)


</td>

<td>

![Screenshot from 2023-06-10 13-27-20](https://github.com/JhonatanErosa/Project-2_Portfolio/assets/66476329/0b9964d1-36d3-46d0-b679-abce7a069bb9)


</td>
</tr>
</table>


#### Requirement

![image](https://github.com/sinruidotec/Portfolio-part-2-Microverse/assets/66476329/641d0c3f-f77f-4683-a5d1-b0c8780bdbe6)